src/funfuzz/autobisectjs/autobisectjs.py

```diff
@@ -38,6 +38,9 @@
     from pathlib import Path  # pylint: disable=import-error
     import subprocess
 
+AUTOBISECTJS_LOG = logging.getLogger("autobisectjs")
```

Comment:
For brevity you could just call this `LOG`. Up to you.

Reply:
I think I'd prefer to keep it as `AUTOBISECTJS_LOG`.